README.md

```diff
@@ -2,207 +2,436 @@
+
+
+* [Homebrew](https://docs.brew.sh/Installation)
+* [Git](https://formulae.brew.sh/formula/git#default)
+* [Php](https://formulae.brew.sh/formula/php#default)
+* [Composer](https://formulae.brew.sh/formula/composer#default)
+* [Docket Desktop](https://docs.docker.com/desktop/install/mac-install/)
+* [Drupal](https://www.digitalocean.com/community/tutorials/how-to-develop-a-drupal-9-website-on-your-local-machine-using-docker-and-ddev#step-1-mdash-installing-ddev)
+* [jq](https://formulae.brew.sh/formula/jq#default)
+* [npm](https://radixweb.com/blog/installing-npm-and-nodejs-on-windows-and-mac)
+* [Visual Studio Code](https://code.visualstudio.com/download)
+
+[back to top](#usagov-2021)
+
+## Setting Up the Project
 
-```
-bin/init
-docker compose up
-```
+### Initialization
 
-## Full Project Setup
-### Download SQL Database
-Safe development database dumps are kept in Google Drive. You can download and import a SQL database from https://drive.google.com/drive/folders/1zVDr7dxzIa3tPsdxCb0FOXNvIFz96dNx?usp=sharing. We recommend using the latest database available.
 
-Unzip the file and insert directly into the **root** directory.
+Follow these steps when you first start working on the project, or anytime you want to reset your local development environment:
+
 
-### Initialization
 **Note: Please wait until each command finishes before running the next. Expect long wait times. We recommend keeping your laptop (if you're using one) plugged in during this setup.**
 
-1. Open up your IDE/terminal and run the following commands.
+
+1. Open a terminal and go to the project folder.
+
+
+2. Run the these commands in the terminal:
+
+
 ```
 bin/init
 docker compose up
 ```
 
+
 Wait until messages stop scrolling by; the final message will probably be a message from node saying "Starting 'watch-sass' ..."
 
-2. Head to `localhost` (no port number needed) in your respective browser. Initially, this will show an empty Drupal site. 
 
-Web logging from "cms" should appear in your terminal as the request is served. This can take a minute to get started.
+3. Open your browser and go to `localhost` (no port number needed). Initially, this will show an empty Drupal site.
+**Note:** You should see logging messages in the terminal as the sote loads. This may take a minute.
+
+
+[back to top](#usagov-2021)
+
+
+### Database Setup
+
+
+Once you finish the previous section, follow these steps to set up your USAgov database:
+
+
+1. Download the latest SQL database available from [our Google Drive](https://drive.google.com/drive/folders/1zVDr7dxzIa3tPsdxCb0FOXNvIFz96dNx?usp=sharing).
+
+
+2. Unzip the file and move the .sql file directly into the **root** of your USAgov project folder.
+
+
+3. Open a new terminal and go to your USAgov project folder.
+
+
+4. Run this command to populate the database from the SQL file you downloaded:
```

Comment:
Does there need to be a step here about renaming the SQL file or am I crazy?

Reply:
You're right! You don't need to change the title but if you don't change it, the command is different. I change that section. Thank you!